pyproject.toml

```diff
@@ -39,34 +39,31 @@ ignore = [
     "TD", # flake8-todos
     "PGH003", # blanket-type-ignore
     "E501", # line-too-long
+    "PLE2513", # invalid-character-esc
+    "PTH", # flake8-use-pathlib
 
-    # if-else blocks that could be shortened (https://docs.astral.sh/ruff/rules/if-else-block-instead-of-if-exp/)
-    "SIM108",
-    # Allow for returns in try blocks
-    "TRY300",
-    # Abstract raise to an inner function
-    "TRY301",
-     # Avoid specifying long messages outside the exception class
-    "TRY003",
-    # Too many statements, too many branches, too many arguments, too many returns
-    "PLR0912", "PLR0915", "PLR0913", "PLR0911",
-    # Magic value used in comparison (foo == 5)
-    "PLR2004",
-    # Explicit string concatenation
-    "ISC003",
-    # Avoid using the generic variable name `df` for DataFrames
-    "PD901",
-    # Replace legacy `np.random.*` call with `np.random.Generator`
-    "NPY002",
+    # These rules were discussed and disabled
```

Comment:
Nit, consider alphabetizing everything in `ignore` (within their respective sections).

Reply:
Resolved in [f5eb618](https://github.com/PrairieLearn/PrairieLearn/pull/11154/commits/f5eb61897f7bd28b61ce798bfd23cb80cb386714).